examples/pytest-app-factory/Dockerfile

```diff
@@ -5,7 +5,8 @@ RUN pip install flask pytest
 ADD . /tmp/latest
 RUN pip install -e /tmp/latest --upgrade
 
-ENV prometheus_multiproc_dir=/tmp
+ENV PROMETHEUS_MULTIPROC_DIR=/tmp
```

Comment:
This could also go without the equals sign, but I can fix it up afterwards.

Reply:
```suggestion
ENV PROMETHEUS_MULTIPROC_DIR /tmp
```

Reply:
Oh yeah, not sure how that got in